### PR TITLE
BIA-655: Add Postgres tests: rls_StudentDataAuthorization View

### PR DESCRIPTION
**Pull request by nsd-daniel**
- Adding PostgreSQL data insertion and column dictionary
- Changing dimension to accept local column dictionary

### CODE REVIEW

File: src/EdFi.AnalyticsMiddleTier.Tests/TestCases/StudentDataAuthorization/PostgreSQL/v_3_2/0001_view_should_match_column_dictionary.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "ColumnName": "sectionid",
+      "DataType": "character varying"
+    },
+    {
+      "ColumnName": "begindate",
+      "DataType": "timestamp without time zone"
+    },
+    {
+      "ColumnName": "enddate",
+      "DataType": "timestamp without time zone"
```
Comment (jleiva-gap) [inline src/EdFi.AnalyticsMiddleTier.Tests/TestCases/StudentDataAuthorization/PostgreSQL/v_3_2/0001_view_should_match_column_dictionary.json:23]: "DataType" : "date"

File: src/EdFi.AnalyticsMiddleTier.Tests/TestCases/StudentDataAuthorization/PostgreSQL/v_3_2/0001_view_should_match_column_dictionary.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "ColumnName": "schoolkey",
+      "DataType": "character varying"
+    },
+    {
+      "ColumnName": "sectionid",
+      "DataType": "character varying"
+    },
+    {
+      "ColumnName": "begindate",
+      "DataType": "timestamp without time zone"
```
Comment (jleiva-gap) [inline src/EdFi.AnalyticsMiddleTier.Tests/TestCases/StudentDataAuthorization/PostgreSQL/v_3_2/0001_view_should_match_column_dictionary.json:19]: "DataType" : "date"

**Review by jleiva-gap — CHANGES_REQUESTED** [review]
1. Postgres tests are not running. You must update the class to inherit from the correct class.

public abstract class When_querying_the_StudentDataAuthorization_view : When_querying_a_view_postgres

2. You must update the conditions to add quotes to the StudentKey and the SchoolKey. In postgres it takes them as varchar and the query will fail.
3. The data type for the begindate and enddate columns is incorrect. According to the query it should be date.

File: src/EdFi.AnalyticsMiddleTier.Tests/TestCases/StudentDataAuthorization/PostgreSQL/v_3_2/0001_view_should_match_column_dictionary.json (hunk 1 above)
Comment (nsd-daniel) [reply src/EdFi.AnalyticsMiddleTier.Tests/TestCases/StudentDataAuthorization/PostgreSQL/v_3_2/0001_view_should_match_column_dictionary.json]: changes were done

**Review by jleiva-gap — APPROVED** [review]

**Pull request merged**